### ISSUE TITLE
Add audio support

### ISSUE THREAD

**Issue by Fossean1** [defold/extension-videoplayer-mpeg#8]

**Comment by padreputativo** [comment]
Would this new update allow adding sound to the player?

https://defold.com/manuals/sound-streaming/

**Comment by JCash** [comment]
No, but the idea is that this one will add the possibility to do so:
https://github.com/defold/defold/issues/10169
